test/e2e/apps/statefulset.go

```diff
@@ -1124,6 +1124,22 @@ var _ = SIGDescribe("StatefulSet", func() {
 			appTester.run()
 		})
 	})
+	// Make sure minReadySeconds is honored
+	// Don't mark it as conformance yet
+	ginkgo.It("MinReadySeconds should be honored when enabled [Feature:StatefulSetMinReadySeconds] [alpha]", func() {
+		ssName := "test-ss"
+		headlessSvcName := "test"
+		// Define StatefulSet Labels
+		ssPodLabels := map[string]string{
+			"name": "sample-pod",
+			"pod":  WebserverImageName,
+		}
+		ss := e2estatefulset.NewStatefulSet(ssName, ns, headlessSvcName, 1, nil, nil, ssPodLabels)
+		setHTTPProbe(ss)
+		ss, err := c.AppsV1().StatefulSets(ns).Create(context.TODO(), ss, metav1.CreateOptions{})
+		framework.ExpectNoError(err)
+		e2estatefulset.WaitForStatusAvailableReplicas(c, ss, 1)
+	})
```

Comment:
should we also test the minReadySeconds time itself in another test?

Reply:
minReadySeconds may have issue because of clockskew between master and worker nodes

Reply:
What is the purpose of this additional test?
I cannot find any minReadySeconds specific part in that.
If that could not be stable, it would be better to write the specific part as commented-out.

Reply:
This test ensures available replicas are matching what we have spec. minReadySeconds is the amount of time STS controller waits before marking a pod ready.